<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen</class>
 <widget class="QWidget" name="Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_workDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_select">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_play">
     <property name="text">
      <string>play</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
